spec/views/bundle_context_spec.rb

```diff
@@ -1,9 +1,12 @@
-require 'rails_helper'
-
 RSpec.describe 'bundle_context/index' do
-  it 'displays the form correctly' do
-    # Should render the test page
-    render
-    expect(rendered).to match(/Project/)
+
+  context "Displays the Bundle Context Form" do
+    
+    it 'displays the form correctly' do
+      assign(:bundle_context, BundleContext.new)
+      # Should render the test page
+      render
+      expect(rendered).to match(/Project/)
+    end
```

Comment:
is this where there should be tests for incorrect data re-displaying, and appropriate error messages in the right place, etc?

Reply:
Tests for errors in BundleContext view now added in commit  ea37556